README.md

```diff
@@ -3,29 +3,105 @@
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 This repository holds the source code for the iOS, macOS, tvOS & watchOS versions of Realm Swift & Realm Objective-C.
 
-## Features
-
-* **Mobile-first:** Realm is the first database built from the ground up to run directly inside phones, tablets and wearables.
-* **Simple:** Data is directly [exposed as objects](https://realm.io/docs/objc/latest/#models) and [queryable by code](https://realm.io/docs/objc/latest/#queries), removing the need for ORM's riddled with performance & maintenance issues. Most of our users pick it up intuitively, getting simple apps up & running in minutes.
-* **Modern:** Realm supports relationships, generics, vectorization and Swift.
-* **Fast:** Realm is faster than even raw SQLite on common operations, while maintaining an extremely rich feature set.
+## Why Use Realm
+
+* **Intuitive to Developers:** Realm’s object-oriented data model is simple to learn, doesn’t need an ORM, and lets you write less code.
+* **Designed for Offline Use:** Realm’s local database persists data on-disk, so apps work as well offline as they do online.
+* **Built for Mobile:** Realm is fully-featured, lightweight, and efficiently uses memory, disk space, and battery life.
+
+## Object-Oriented: Streamline Your Code
+
+Realm was built for mobile developers, with simplicity in mind. The idiomatic, object-oriented data model can save you thousands of lines of code.
+
+```swift
+// Define your models like regular Swift classes
+class Dog: Object {
+    @Persisted var name: String
+    @Persisted var age: Int
+}
+class Person: Object {
+    @Persisted(primaryKey: true) var _id: String
+    @Persisted var name: String
+    @Persisted var age: Int
+    // Create relationships by pointing an Object field to another Class
+    @Persisted var dogs: List<Dog>
+}
+// Use them like regular Swift objects
+let dog = Dog()
+dog.name = "Rex"
+dog.age = 1
+print("name of dog: \(dog.name)")
+
+// Get the default Realm
+let realm = try! Realm()
+// Persist your data easily with a write transaction 
+try! realm.write {
+    realm.add(dog)
+}
+```
+## Live Objects: Build Reactive Apps
+Realm’s live objects mean data updated anywhere is automatically updated everywhere. If you need to freeze elements of your data for Reactive frameworks, you can do that, too.
+```swift
+// Open the default realm.
+let realm = try! Realm()
+
+var token: NotificationToken?
+
+let dog = Dog()
+dog.name = "Max"
+
+// Create a dog in the realm.
+try! realm.write {
+    realm.add(dog)
+}
+
+//  Set up the listener & Observe object notifications.
+token = dog.observe { change in
+    switch change {
+    case .change(let properties):
+        for property in properties {
+            print("Property '(property.name)' changed to '(property.newValue!)'");
```

Comment:
Did the \ get stripped? Or is it not needed?
```suggestion
            print("Property '\(property.name)' changed to '\(property.newValue!)'");
```

Reply:
Think it's wrong on `realm.io`, where I copy and pasted from.